.scalafmt.conf

```diff
@@ -0,0 +1,52 @@
+version = 2.2.2
```

Comment:
Using the latest version. This is the only difference regarding Akka's configuration.